azure-mgmt-cdn/src/main/java/com/microsoft/azure/management/cdn/CdnEndpoint.java

```diff
@@ -0,0 +1,445 @@
+package com.microsoft.azure.management.cdn;
+
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.cdn.implementation.EndpointInner;
+import com.microsoft.azure.management.resources.fluentcore.arm.CountryISOCode;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.ExternalChildResource;
+import com.microsoft.azure.management.resources.fluentcore.model.Settable;
+import com.microsoft.azure.management.resources.fluentcore.model.Wrapper;
+
+import java.util.List;
+
+/**
+ * An immutable client-side representation of an Azure CDN endpoint.
+ */
+@Fluent
+public interface CdnEndpoint extends
+        ExternalChildResource<CdnEndpoint, CdnProfile>,
+        Wrapper<EndpointInner> {
+
+    /**
+     * Get the originHostHeader value.
```

Comment:
We use the 3rd person declarative mood for member descriptions, so "Gets" not "Get", "Specifies" not "Specify", etc. 
pls scrub the Javadocs to get them consistent with that. thx

Btw - on parameters properties returning a value, we don't need the description,  just the @return, so:
@return origin host header